content/bn/docs/reference/glossary/controller.md

```diff
@@ -0,0 +1,29 @@
+---
+title: কন্ট্রোলার
+id: কন্ট্রোলার
+date: ২০১৮-০৪-১২
+full_link: /docs/concepts/architecture/controller/
```

Comment:
Use `full_link: /bn/docs/concepts/architecture/controller/` as we are localizing in the `bn` directory.